fixture-monkey/src/test/java/com/navercorp/fixturemonkey/test/InnerSpecTest.java

```diff
@@ -827,4 +828,18 @@ void setNotNull() {
 
 		then(actual).isNotNull();
 	}
+
+	@Property
+	void keysForCollection() {
+		List<String> keyList = Arrays.asList("key1", "key2", "key3");
+		Map<String, String> actual = SUT.giveMeBuilder(MapObject.class)
+			.setInner(
+				new InnerSpec()
+					.property("strMap", it -> it.keys(keyList).size(3))
+			)
+			.sample()
+			.getStrMap();
+
+		then(actual.keySet()).contains("key1", "key2", "key3");
```

Comment:
You can also use `containsAll' instead of `contains`, which makes the code shorter.